data/queries.graphql

```diff
@@ -27,6 +27,7 @@ type Query {
 
   dataset(where: DatasetWhereUniqueInput!): Dataset!
   datasets(where: DatasetWhereInput, first: Int, skip: Int): [Dataset!]!
+  searchDataset(where: DatasetWhereUniqueInput!): Dataset
```

Comment:
Usually, the signature of a search would be more something like 
`searchDataset(where: DatasetWhereInput!): [Dataset!]!`

I agree that for our use case this is not needed, but I think it might be better to make it more "standard". What do you think?

Reply:
Yes indeed

Reply:
Yes maybe it's a good thing to do, although in our case `DatasetWhereInput` is used to list the datasets of a workspace:

```graphql
input DatasetWhereInput {
  workspaceSlug: String!
}
```

So if we do this it complexifies a bit the queries using `DatasetWhereInput ` since you might want to filter by workspace + by name

Reply:
Ah yes indeed it's a unique input so no need to have an array. but the "search" name is a bit misleading I guess.